pkg/kritis/review/validating_transport.go

```diff
@@ -69,13 +69,17 @@ func (avt *AttestorValidatingTransport) GetValidatedAttestations(image string) (
 		return nil, err
 	}
 	for _, a := range attestations {
-		decoded_sig, err := base64.StdEncoding.DecodeString(a.Signature)
+		// TODO(acamadeo): Temporarily hardcoding this to work only for PGP Attestations.
```

Comment:
But return an error in case it's not a PGP Attestation (or if mismatch between type and payload).

Reply:
Moreover, I suggest turning this into a function (e.g. VerifyPgpSignature) immediately.  Later we'll add VerifyPkixSignature and decide which one to call depending on the type of the RawAttestation.

Reply:
Done.